EpiAware/src/epimodel.jl

```diff
@@ -1,24 +1,24 @@
 abstract type AbstractEpiModel end
 
 
+
 """
-    struct EpiModel{T}
+    struct EpiModel{T<:Real} <: AbstractEpiModel
 
-The `EpiModel` struct represents a basic renewal model.
+EpiModel represents an epidemiological model with generation intervals, delay intervals, and observation delay kernel.
```

Comment:
Can we at some point give this a cooler name?

Reply:
Yes!

Reply:
Lets do this in a follow on refactor